src-python/drbench/drbench/drbench.py

```diff
@@ -45,6 +56,7 @@ def choice_to_mode(c):
 
 
 @cli.command()
+@click.option('--dontplot', is_flag=True, help='just create json files')
```

Comment:
```suggestion
@click.option('--plot', is_flag=True, default=True, show_default=True, help='generate plots')
```

Reply:
My intention was that common use case is to run analysis and plotting together, so extra option `--dontplot` is added occasionally.

I understand you want the user be forced to write '--plot' when he/she wants plots to be generated just after analysis. Adding `default=True, show_default=True` is noop, has the same effect as not writing it.
Please confirm this is what you want and I will change `--dontplot` to `--plot`.

Reply:
The behavior I want is:
```
dr-bench
```
does the plot and:
```
dr-bench --plot=false
```
Does not plot. It is not common to have negative flags, usually default=true and explicitly disabled. I did not check if `--plot=false` is the proper way to disable a `default=True` flag. The `show_default=True` puts the default value in the help message

Reply:
I was wrong. Here is the proper way to do it:
https://stackoverflow.com/a/76182645/2525421